tendrl/commons/objects/node_context/__init__.py

```diff
@@ -25,10 +25,13 @@ def __init__(self, node_id=None, fqdn=None,
         self.node_id = node_id or self._get_node_id() or self._create_node_id()
         self.fqdn = fqdn or socket.getfqdn()
 
-        curr_tags = []
+        curr_tags = ''
```

Comment:
While starting node-agent, the tags loading used to fail. Will remove and check what was the actual impact.